helix-term/src/commands/typed.rs

```diff
@@ -1783,6 +1783,54 @@ fn run_shell_command(
     Ok(())
 }
 
+fn show_keymap(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+    let mode = match args.first().map_or("", |a| a) {
+        "normal" | "n" => Some(Mode::Normal),
+        "insert" | "i" => Some(Mode::Insert),
+        "select" | "s" => Some(Mode::Select),
+        _ => None,
+    };
+
+    let callback = async move {
+        let call: job::Callback = Callback::EditorCompositor(Box::new(
+            move |editor: &mut Editor, compositor: &mut Compositor| {
+                let keymaps = compositor.find::<ui::EditorView>().unwrap().keymaps.map();
+                let mut output = Vec::with_capacity(6);
+                for (_, keymap) in keymaps
+                    .iter()
+                    .filter(|(m, _)| mode.is_none() || mode.as_ref() == Some(*m))
+                {
+                    if let Some(info) = keymap.root().node().map(|node| node.infobox()) {
+                        output.push(format!("{}\n", info.title));
+                        output.push(info.text);
+                    }
+                }
+                let contents = ui::Markdown::new(
+                    format!("```Keys\n{}\n```", output.join("\n")),
```

Comment:
If "Keys" is meant as a header then it should have a newline before it. Currently it's being interpreted as the language of the codefence